### DISCUSSION TITLE
Regarding field OPT-060-Lot for sdk version 1.5.1

### DISCUSSION THREAD

**Discussion by Nitin-789** [OP-TED/eForms-SDK#602] [Q&A] [answered]
Hi,
We see that this field OPT-060-Lot for notice type 17 is a mandatory field in the upgraded UI in Ted portal, but since we are sticking to 1.5.1 for now this field OPT-060-Lot should not be mandatory field in validation right and sending payload without this field should not throw any validation error right ?

Thanks for your help,
Nitin

**Comment by YvesJo** [comment, marked as answer]
Hi,
The field OPT-060-Lot is necessary for the identification of BT-70-Lot, both shall be under the same "_cac:ContractExecutionRequirement_" element, otherwise an error may be raised. Be ware that OPT-060 as an implicit field was not explicitly defined in SDK 1.5 and the corresponding XML fragment has to be generated automatically at XML generation once BT-70 is specified.
Also consider that for a given SDK version (e.g. SDK 1.5) it is always the latest version that has to be used, which in your case is SDK 1.5.3
